File: PHP/card.php
<?php
require_once('payment.php');
class Card extends payment{
    public $id;
    public $number;
    public $cvv;
    public $date;

    public function __construct($id, $number){
        //Acceso a la super clase
        parent::__construct($id);
        $this->number = $number;
    }
}

?>

File: JAVA/Card.java
import java.sql.Date;

public class Card extends Payment{

    // Propiedades
    Integer id;
    Integer number;
    Integer cvv;
    Date date;

    // Método constructor
    public Card(Integer id, Integer number){
        super(id);
        this.number = number;

    }
}
